src/schedulers/marrainageScheduler.js

```diff
@@ -26,6 +28,19 @@ const reloadMarrainages = async function () {
     .catch(console.error);
 };
 
+module.exports.removeMarrainageForExpiredUsers = async () => {
+  const users = await BetaGouv.usersInfos();
+  const expiredUsers = utils.getExpiredUsersForXDays(users, 1);
+
+  expiredUsers.forEach(async (user) => {
+    await knex('marrainage')
+    .where({ last_onboarder: user.id })
+    .update({ completed: false });
+    console.log(`Le marrainage de ${user.fullname} a pris fin suite à son départ.`);
```

Comment:
Je pense qu' un message plus explicite pourrait être pas mal pour comprendre que user.fullname est le nom du marrain et non du marrainé
```suggestion
    console.log(`${user.fullname} a été retiré.e de son role de marrain.e suite à son départ.`);
```